v3/src/models/formula/functions/date-functions.test.ts

```diff
@@ -0,0 +1,56 @@
+import { formatDate } from "../../../utilities/date-utils"
+import { math } from "./math"
+
+describe("date", () => {
+  it("returns current date if no arguments are provided", () => {
+    const fn = math.compile("date()")
+    expect(fn.evaluate()).toEqual(formatDate(new Date()))
+  })
+
+  it("interprets a small number as 20xx year if it's smaller than current year + 10", () => {
```

Comment:
In the date-parser's `fixYear` function, we use:
```typescript
function fixYear(y: string) {
    if (y.length === 2) {
      const yNumber = Number(y)
      if (yNumber < 49) {
        return 2000 + yNumber
      } else {
        return 1900 + yNumber
      }
    }
```
Shouldn't these two assumptions be synchronized? Perhaps current year + 20 or 25?